src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
+    carInputValidation(CARS_NAMES);
+
+    MissionUtils.Console.print(MESSAGES.ANSWER_COUNT);
+    const COUNT = await MissionUtils.Console.readLineAsync("");
+    countInputValidation(COUNT);
+
+    //게임 진행
+    MissionUtils.Console.print("\n" + MESSAGES.RESULT);
```

Comment:
```suggestion
    MissionUtils.Console.print("\n" + MESSAGES.RESULT);
```
오 저는 `MESSAGES.RESULT`에 개행 문자(`\n`)를 포함했었는데 소은님이 해주신 방식도 좋은 것 같네요👍👏💪